content/en/docs/reference/kubectl/kuberc.md

```diff
@@ -12,8 +12,8 @@ such as default options and command aliases. Unlike the kubeconfig file, a `kube
 configuration file does **not** contain cluster details, usernames or passwords.
 
 The default location of this configuration file is `$HOME/.kube/kuberc`.
-You can instruct `kubectl` to look at a custom path for this configuration using
-the `--kuberc` command line argument.
+Use `--kuberc` flag for `kubectl` or `KUBERC` environment variable to set path
```

Comment:
Consider the following suggestion:

You can instruct `kubectl` to look at a custom path for this configuration file by using the `--kuberc` command line argument or by setting the `KUBERC` environment variable.

Reply:
Considered and declining. Too long and not straight to the point, too much useless text.

Reply:
If you want something shorter, try

"To provide `kubectl` with a custom path to a `kuberc` file, use the `--kuberc` flag or set the `KUBERC` environment variable."